lib/video-moq/video-moq.css

```diff
@@ -2,14 +2,32 @@
  each element in dev tools and cleaned up some classes.
  Feel free to modify. */
 
+#base {
+	position: relative;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+}
+
+#base.pip-mode {
+	background-color: rgb(0, 0, 0);
+	height: 100%;
+}
+
 canvas#canvas {
 	z-index: 0;
 	background: rgb(28, 28, 28);
+	object-fit: contain;
 }
 
 #controls {
 	z-index: 10;
 }
+
+#controls:focus-within {
+	opacity: 1;
+}
```

Comment:
Prevents the controls from hiding when a button from within the control bar receives focus.